### ISSUE TITLE
No Downgrade possible ???

### ISSUE THREAD

**Issue by Toddi3D** [CrealityOfficial/Creality_Sonic_Pad#35]
It would be great if we can do a Downgrade to previous Firmware.

**Comment by Creality-Ralf** [comment]
What specific problems did you have that required a downgrade? In fact, we have released a USB burner package to recover sonicpad that has crashed for some reason

**Comment by TheJadeGeek** [comment]
Unable to level when bed is heated, get probe error or end stop error after a few probes. 

setting Z Offset appears to do nothing, my bed is now scratched all to hell. 

just a couple issues with the most recent update that would warrant wanting to rollback a version.

**Comment by Creality-Ralf** [comment]
> Unable to level when bed is heated, get probe error or end stop error after a few probes.
> 
> setting Z Offset appears to do nothing, my bed is now scratched all to hell.
> 
> just a couple issues with the most recent update that would warrant wanting to rollback a version.

Try using this: https://github.com/CrealityOfficial/Creality_Sonic_Pad_Firmware

**Comment by TheJadeGeek** [comment]
Yeah I eventually got that to work. After lots of help. Found out that my issue is a factory defect with my cr10 smart and a factory spliced cable failed. On May 28, 2023, at 19:35, Luo52 ***@***.***> wrote:﻿

Unable to level when bed is heated, get probe error or end stop error after a few probes.
setting Z Offset appears to do nothing, my bed is now scratched all to hell.
just a couple issues with the most recent update that would warrant wanting to rollback a version.

Try using this: https://github.com/CrealityOfficial/Creality_Sonic_Pad_Firmware
